README.md

```diff
@@ -84,8 +84,15 @@ user to focus on the functionality, not on technical details.
 
 ## Naming things
 
-* All YAML or Python files, variables, arguments, repositories, and other such names should follow
-  standard Python naming conventions of being in snake\_case\_naming\_schemes.
+* All YAML or Python files, variables, arguments, repositories, and other such names (like
```

Comment:
Does `arguments` here mean Python function/method named arguments or kwargs names?

Reply:
This was inherited from the original OASIS document. @greg-hellings @robled what did you mean back then?

Reply:
regardless of the answer to the question, this change lgtm

Reply:
I believe that the original intent was that it does include things like Python functions, etc.